src/core/etl/tests/Flow/ETL/Tests/Unit/Row/Schema/Formatter/ASCIISchemaFormatterTest.php

```diff
@@ -34,12 +43,12 @@ public function test_format_nested_schema() : void
             <<<SCHEMA
 schema
 |-- user: Flow\ETL\Row\Entry\StructureEntry (nullable = false)
-|    |-- name: [Flow\ETL\Row\Entry\StringEntry, Flow\ETL\Row\Entry\NullEntry] (nullable = true)
-|    |-- age: Flow\ETL\Row\Entry\IntegerEntry (nullable = false)
-|    |-- address: Flow\ETL\Row\Entry\StructureEntry (nullable = false)
-|        |-- street: [Flow\ETL\Row\Entry\StringEntry, Flow\ETL\Row\Entry\NullEntry] (nullable = true)
-|        |-- city: [Flow\ETL\Row\Entry\StringEntry, Flow\ETL\Row\Entry\NullEntry] (nullable = true)
-|        |-- country: [Flow\ETL\Row\Entry\StringEntry, Flow\ETL\Row\Entry\NullEntry] (nullable = true)
+|    |-- name: ?string (nullable = true)
```

Comment:
Since we can't use FQCN of Entry, schema should be modified to follow the same convention. To make it easier, we can additional method to `Entry` interface, `Entry::phpType() : Type` and then just use toString() on that type which should be aligned also with psalm/phpstan types, maybe except list/map/structure but those are anyway logical types.